pkg/util/docker/dockerfile/dockerfile_test.go

```diff
@@ -94,7 +94,7 @@ WORKDIR /home
 			t.Errorf("%s: parse error: %v", name, err)
 			continue
 		}
-		got := ParseTreeToDockerfile(node)
+		got := ParseTreeToDockerfile(node.AST)
```

Comment:
@smarterclayton check pls, this moved in rebase...

Reply:
This is fine, I've made that move myself several times already.